src/pages/ProgramsList/ProgramsMain.js

```diff
@@ -0,0 +1,74 @@
+import React, { Fragment } from "react";
+import { makeStyles } from "@material-ui/core/styles";
+import { BookOutlined, HomeOutlined } from "@ant-design/icons";
+import { Space, Layout, Menu, Divider } from "antd";
+
+import AppAppBar from "../../modules/views/AppAppBar";
+import AppFooter from "../../modules/views/AppFooter";
+import ProgramsItems from "./ProgramItems";
+
+const { SubMenu } = Menu;
+const { Sider } = Layout;
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    display: "flex",
+    justifyContent: "space-around",
+    backgroundColor: theme.palette.background.paper,
+  },
+  gridList: {
+    width: 800,
+  },
+  icon: {
+    color: "rgba(255, 255, 255, 0.54)",
+  },
+  title: {
+    marginLeft: theme.spacing(3),
+    marginTop: theme.spacing(2),
+  },
+}));
+
+const ProgramsMain = () => {
+  const classes = useStyles();
+
+  const sider = (
+    <Sider width={200} className="site-layout-background">
+      <Menu
+        mode="inline"
+        defaultOpenKeys={["sub1"]}
+        style={{ height: "100%", borderRight: 0 }}
+      >
+        <Space className={classes.title}>
+          <h4>FILTER BY</h4>
+          <Divider />
+        </Space>
+        <SubMenu key="sub1" title="Categories" icon={<BookOutlined />}>
+          <Menu.Item key="1">Psychology</Menu.Item>
+          <Menu.Item key="2">Healthcare</Menu.Item>
+          <Menu.Item key="3">Sports</Menu.Item>
+          <Menu.Item key="4">Food</Menu.Item>
```

Comment:
nit: would prefer this inside a loop so we don't have to hardcode the keys. can be done in the next PR :)

Reply:
okay! 😄